airflow/ui/src/components/SearchDags/SearchDagsButton.tsx

```diff
@@ -31,10 +32,19 @@ export const SearchDagsButton = () => {
     setIsOpen(false);
   };
 
+  useHotkeys(
+    "mod+k",
+    () => {
+      setIsOpen(true);
+    },
+    [isOpen],
+    { preventDefault: true },
+  );
+
   return (
     <Box>
       <Button justifyContent="flex-start" onClick={() => setIsOpen(true)} variant="subtle" w={200}>
-        <MdSearch /> Search Dags
+        <MdSearch /> Search Dags <Kbd size="lg">⌘K</Kbd>
```

Comment:
This is the Apple specific hotkey, can you also (or primarily) document the "normal non Apple" key, e.g. CTRL)?

Reply:
I am using "mod+k" which should map to "ctrl+k" in Linux. I tested this in Ubuntu 20.04 with Firefox 132 which opens only the search bar and not the browser search since `preventMode` is passed as True. This is inline with the Chakra docs that use the Mac specific key notation as per the initial design discussion of the search button but I am open to change.

![image](https://github.com/user-attachments/assets/145ff671-e545-4eeb-8c99-a7a409364640)

https://www.chakra-ui.com/docs/components/kbd#function-keys

https://react-hotkeys-hook.vercel.app/docs/documentation/useHotkeys/basic-usage#modifiers--special-keys

> mod (which listens for ctrl on Windows/Linux and cmd on macOS)

Reply:
Aaah, my bad I was looking at the "wrong" dialog, the DAG list view. The new control is on the DAG detail view. Can report... the problem was roughly 50cm in front of the keyboard. Function is working.

Will you add the same to DAG list view in another PR or can you make it consistent within the same PR?

Reply:
Thanks for testing. For dag list view I think it will be taking the focus to the "search dags" input box and not opening the dialog. I can take it on another PR since I am working on few other things to not block this.

Reply:
Well we don't need to know every OS. We only need to detect a Mac. And since this is only to change between displaying cmd vs ctrl. I think just checking if the UserAgent string includes "Mac" is sufficient. Accidentally showing ctrl is pretty low risk.

Reply:
`userAgent` seems to be also not recommended. Agreed on "Ctrl" which will have more users. I have added a small util function for this which should default to "Ctrl" unless "Mac" is present in `appVersion` .

https://stackoverflow.com/questions/34199584/is-window-navigator-useragent-really-going-to-be-deprecated
https://developer.mozilla.org/en-US/docs/Web/API/Navigator/userAgent

Reply:
Cool!